diff --git a/.bazelrc b/.bazelrc
index d58eafb1b3e..0cddb928342 100644
--- a/.bazelrc
+++ b/.bazelrc
@@ -295,7 +295,7 @@ build:rocm_base --repo_env TF_NEED_ROCM=1
 
 build:rocm --config=rocm_base
 build:rocm --config=release_cpu_linux_base
-build:rocm --action_env=CLANG_COMPILER_PATH="/usr/lib/llvm-17/bin/clang"
+build:rocm --action_env=CLANG_COMPILER_PATH
 build:rocm --action_env=TF_ROCM_CLANG="1"
 build:rocm --linkopt="-fuse-ld=lld"
 # We have some invalid linker scripts in the build,
@@ -655,9 +655,9 @@ build:release_linux_base --action_env PYTHON_BIN_PATH="/usr/bin/python3"
 build:release_linux_base --action_env PYTHON_LIB_PATH="/usr/lib/tf_python"
 build:release_linux_base --python_path="/usr/bin/python3"
 # Set Clang as compiler. Use the actual path to clang installed in container.
-build:release_cpu_linux_base --repo_env=CC="/usr/lib/llvm-17/bin/clang"
-build:release_cpu_linux_base --repo_env=BAZEL_COMPILER="/usr/lib/llvm-17/bin/clang"
-build:release_cpu_linux_base --action_env=CLANG_COMPILER_PATH="/usr/lib/llvm-17/bin/clang"
+build:release_cpu_linux_base --repo_env=CC
+build:release_cpu_linux_base --repo_env=BAZEL_COMPILER
+build:release_cpu_linux_base --action_env=CLANG_COMPILER_PATH
 build:release_cpu_linux_base --linkopt="-fuse-ld=lld"
 # Test-related settings below this point.
 test:release_linux_base --build_tests_only --keep_going --test_output=errors --verbose_failures=true
